rosrust/src/time.rs

```diff
@@ -1,4 +1,6 @@
 use std::cmp;
+use std::convert::From;
```

Comment:
Is this line required? I don't remember I ever wrote it, and I used `From` a lot.